Core/include/Acts/Propagator/AtlasStepper.hpp

```diff
@@ -1224,7 +1224,7 @@ class AtlasStepper {
            std::abs((C1 + C6) - (C3 + C4)));
       if (std::abs(EST) > std::abs(state.options.tolerance)) {
         h = h * .5;
-        state.stepping.stepSize.setValue(h);
+        state.stepping.stepSize.setValue(h * state.stepping.navDir);
```

Comment:
I guess this is to remove any sign again?

Reply:
yeah exactly this is supposed to drop the sign we applied before again